calico-enterprise/reference/resources/networkpolicy.mdx

```diff
@@ -132,8 +134,29 @@ spec:
 
 <Servicematch />
 
+## Application layer policy
+
+Application layer policy is an optional feature of {{prodname}} and
+[must be enabled](../../network-policy/istio/app-layer-policy.mdx)
+in order to use the following match criteria.
+
+:::note
+
+Application layer policy match criteria are supported with the following restrictions.
+
+- Only ingress policy is supported. Egress policy must not contain any application layer policy match clauses.
+- Rules must have the action `Allow` if they contain application layer policy match clauses.
+
+:::
+
+### HTTPMatch
+
+<Httpmatch />
+
+
 ## Supported operations
 
 | Datastore type           | Create/Delete | Update | Get/List | Notes |
 | ------------------------ | ------------- | ------ | -------- | ----- |
+| etcdv3                   | Yes           | Yes    | Yes      |
```

Comment:
I'm not sure what context why supported operations per datastore type are displayed here? In any case, I think you can leave etcdv3 out of this list

Reply:
@electricjesus Can I delete it from Global Network Policy also?